### PR TITLE
Set error message for `duplicate-task-headers` rule

### LINKED ISSUE #4: Warn on duplicate key in custom headers
In Camunda 8/Zeebe one can define custom headers for
* Service Tasks
* User Tasks
* Send Tasks
* Script Tasks
* Message Intermediate Throw Events

If the custom headers contain entries with duplicate keys, this is an error and will be rejected on deployment.

image

Related to https://github.com/camunda/zeebe/issues/9416

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Requires https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/41
Closes #4 

Note this is added for all Camunda 8 versions, as it has been failing on deployment since the first supported Zeebe version (cf. https://github.com/camunda/linting/issues/4#issuecomment-1214751256).

----

image

### CODE REVIEW

File: lib/utils/error-messages.js [hunk 1]
```diff
@@ -77,22 +78,25 @@ function getElementTypeNotAllowedErrorMessage(report, executionPlatformLabel) {
...
   const {
-    duplicateValue,
     node,
-    parentNode
+    parentNode,
+    duplicatedPropertyValue,
+    properties
   } = error;
 
+  console.log(error);
```
Comment (pinussilvestrus) [inline lib/utils/error-messages.js:94]:
```suggestion
```

File: lib/utils/properties-panel.js [hunk 2]
```diff
@@ -167,6 +185,16 @@ export function getErrorMessage(id) {
   if (id === 'formConfiguration') {
     return 'Form JSON configuration must be defined.';
   }
+
+  if (id === 'formConfiguration') {
+    return 'Form JSON configuration must be defined.';
+  }
+
+  console.log('foobar')
```
Comment (pinussilvestrus) [inline lib/utils/properties-panel.js:193]:
```suggestion
```

**Review by pinussilvestrus — COMMENTED** [review]
Thanks for the initiative! Looks really promising, I will take a deeper look on this one + the rules PR once I'm back 👍

**Comment by philippfromme** [comment]
I made a proposal to refactor this before we merge it (cf. https://github.com/camunda/linting/pull/12/commits/33d0381a2aa524e55e261f8247f8e8910b0e7f02). @pinussilvestrus will have a look at the proposal and we'll squash before merging.

**Comment by pinussilvestrus** [comment]
✅ from my side 👍 (cf. https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/41#pullrequestreview-1076892991).

Let's merge and release the rule and bring this out of the door.

**Review by philippfromme — APPROVED** [review]
✅

**Pull request merged**